src/main/java/de/zalando/aruha/nakadi/controller/EventStreamController.java

```diff
@@ -173,11 +173,23 @@ public StreamingResponseBody streamEvents(
         };
     }
 
+    private String invalidCursorMessage(final CursorError error, final Cursor cursor) {
+        switch (error) {
+            case PARTITION_NOT_FOUND:
+                return "non existing partition " + cursor.getPartition();
+            case EMPTY_PARTITION:
+                return "partition " + cursor.getPartition() + " is empty";
+            case UNAVAILABLE:
+                return "offset " + cursor.getOffset() + " for partition " + cursor.getPartition() + " unavailable";
```

Comment:
I think I don't really get your point. Probably it's just too complicated for me :)
My point is that `unavailable` is an adjective and for me `offset 5482 for partition 2 unavailable` sounds the same as for example `partition 3 empty` or `rose red`